styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
+  text-align: justify;
```

Comment:
would it be better practice to give it a column? I feel like it's very microsoft word looking right now....